src/pages/TranslationsPage/TranslationForm.jsx

```diff
@@ -0,0 +1,78 @@
+import React, {Component} from "react";
+import {Form, Button} from 'react-bootstrap'
+
+import {Header} from "../../components/Header";
+import {Footer} from "../../components/Footer";
+
+class TranslationForm extends Component{
+      constructor(props){
+        super(props)
+        this.state ={
+            track_id:'',
+            text:'',
+            language:'en',
+            auto_translate:"true"
+        }
+    }
+
+    changeHandler = (e) =>{
+        this.setState({[e.target.name]: e.target.value})
+    }
+
+    submitHandler = e => {
+        e.preventDefault()
+        this.state.track_id = this.props.location.state
+        this.state.auto_translate = (this.state.auto_translate==="true")
+        const requestOptions = {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify(this.state)
+        };
+        console.log(JSON.stringify(this.state))
+        fetch(`http://127.0.0.1:8000/api/tracks/${this.props.location.state}/translations/`, requestOptions)
```

Comment:
ideally the hosts shouldn't be hardcoded but come from environment. this could be the subject for the one of the subsequent PRs